README.md

```diff
@@ -37,10 +37,12 @@ You need Linux or macOS. On Windows, we recommend installing the Windows Subsyst
 
 - Download [Homebrew](https://brew.sh/), run `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"` in terminal
 - Install [cairo](https://formulae.brew.sh/formula/cairo#default) via homebrew `brew install cairo`
+    - Locate the cairo folder within the 'include' directory of the installed cairo package. Move it into the vendor directory so that the cairo files are in `vendor/cairo` under the LOST repository
 - Install [Eigen]("https://formulae.brew.sh/formula/eigen#default") via homebrew `brew install eigen`
-    - Locate Eigen files and move them to `vendor/eigen3/Eigen` under the LOST repository
+    - Locate the eigen3 folder within the 'include' directory of the installed Eigen package. Move it into the vendor directory so that the eigen files are in `vendor/eigen3/Eigen` under the LOST repository
 - Install [groff]("https://formulae.brew.sh/formula/groff#default") via homebrew `brew install groff`
 - If you get errors mentioning 'ASAN' or 'AddressSanitizer', try `make clean` and then `make LOST_DISABLE_ASAN=1` to disable ASAN. See the Linux section above for more details.
+- On Apple Silicon, if you an error like `error: linker command failed with exit code 1` or `library 'cairo' not found`, set LIBS in the Makefile to `LIBS = -L/opt/homebrew/lib -lcairo` and add a line `CFLAGS = -I/opt/homebrew/include/cairo` under it.
```

Comment:
a user should never have to modify a makefile or other source code as part of a normal build process.